CONTRIBUTING.md

```diff
@@ -699,7 +699,7 @@ Verify that .proto files and autogenerated code are in sync by running
 
 ##### Database Schema Changes
 
-MLflow's Tracking component supports storing experiment and run data in
+MLflow's Tracking component supports storing experiments and running data in
```

Comment:
I'm not sure if this is a typo—the original seems to be saying that both experiment data and run data can be stored in a SQL backend

Reply:
I think @daniellok-db is right. `running data` sounds strange.